src/overlays/actors/ovl_En_And/z_en_and.c

```diff
@@ -30,22 +29,158 @@ ActorInit En_And_InitVars = {
     (ActorFunc)EnAnd_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_0600B380;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18B90.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18BD8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18C50.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/EnAnd_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/EnAnd_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/EnAnd_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18ED0.s")
+static AnimationInfoS sAnimationInfo[8] = {
+    { &gAndStaticAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
```

Comment:
```suggestion
static AnimationInfoS sAnimationInfo[] = {
    { &gAndStaticAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
    { &gAndIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
```
We avoid putting hard constants in arrays like this so that it's easier to change later